README.md

```diff
@@ -107,27 +257,59 @@ Example request and response dump:
 
 ```
 POST /v1/checks HTTP/1.1
-Host: api.checklyhq.com
+Host: api-test.checklyhq.com
```

Comment:
We shouldn't have the api-test stuff here I guess

Reply:
Hmm, that's a good question. I guess we expect people to be testing their stuff against the test API? So when they turn on debugging this is what they'll see. Or is the test API internal only? Because we expose it in the integration tests, necessarily.

Reply:
The test api is internal only. We should not depend on it

Reply:
Ah, I've misunderstood this. I asked Tim about setting up a Checkly account dedicated to running integration tests, and I understood that he'd told me to use the test API. If we should be using the prod API for integration tests, then I'll need a prod account for it!

Reply:
@tnolet could you supply me with the credentials for the prod API test account, please, and I'll update the PR.

Reply:
@bitfield I invited you to the account we call "Production External Integration" on app.checklyhq.com.

Reply:
@bitfield please update so we can merge the PR and continue with the Terraform part.